pkg/model/dev.go

```diff
@@ -892,7 +892,7 @@ func (dev *Dev) LoadRemote(pubKeyPath string) {
 	p := Secret{
 		LocalPath:  pubKeyPath,
 		RemotePath: authorizedKeysPath,
-		Mode:       0600,
+		Mode:       0644,
```

Comment:
Why this change? Wasn't it already on master?

Reply:
:hushed: some failure updating with master. I will check it. Thanks Nacho!